src/fstext/context-fst-inl.h

```diff
@@ -310,9 +310,9 @@ bool ContextFstImpl<Arc, LabelT>::CreateArc(StateId s,
     // the output arcs, just 0.
     return CreatePhoneOrEpsArc(s, nextstate, olabel, phoneseq, oarc);
   } else {
-    std::cerr << "ContextFst: CreateArc, invalid olabel supplied [confusion about phone list or disambig symbols?]: "<<(olabel);
-    exit(1);
- }
+    KALDI_ERR << "ContextFst: CreateArc, invalid olabel supplied [confusion "
```

Comment:
I am sorry for pulling you into a discussion with a change so insignificant, did not really expect that. I just went back into the file to revert to cerr, and found that the same file uses `KADI_WARN`, `KALDI_ASSERT`, and also C `assert()`. It won't be consistent in this sense either way, KALDI_ERR or `std::cerr` (but at the very least, Kaldi-consistent with KALDI_ERR). But, in general, I think that if you treat fstext as (at least in the ideal) Kaldi-independent piece, it would be better off using OpenFST's LOG(FATAL/ERROR/...) macros than raw `cerr`. They default to stderr, but allow to intercept logging by substituting custom `fst/log.h` (as I do). If you agree, I'd be happy to make a thorough clean changeset to make logging consistent with OpenFST logging macros in this library.